src/lib/directives.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+function device(type){
+  return `'{$STAMP ${type}}`;
+}
+
+function pbasic(ver){
+  return `'{$PBASIC ${ver}}`;
+}
+
+const directives = {
```

Comment:
no point in exporting an object here, just export a method that takes a board and language and returns your directive string.  You are doing interpolation inside `src/plugins/handlers.js` so this module serves no purpose unless it moves that logic into a function instead.